### ISSUE TITLE
The additional DMS assays added in the article has not been released

### ISSUE THREAD

**Issue by Meron-TANG** [OATML-Markslab/ProteinNPT#5]
When I check the data for this rep., I could not find the additional DMS assays added for the extended benchmark record in the paper. These include: A0A220GHA5, CAS9 STRP1 negative, BAC7 (single and multiple mutations) and CAPSD BPMS2, etc. Could you please release your processed version?

**Comment by pascalnotin** [comment]
Hi @Meron-TANG,

We recently released a significantly larger version of the DMS benchmarks within our [ProteinGym paper](https://papers.nips.cc/paper_files/paper/2023/hash/cac723e5ff29f65e3fcbb0739ae91bee-Abstract-Datasets_and_Benchmarks.html) (see also the [repo](https://github.com/OATML-Markslab/ProteinGym) for details on how to access the data). For the most part, the new benchmark subsumes and vastly expands beyond the set of assays we had in the ProteinNPT paper. However, when curating this newer version, we eventually decided to not include a handful of the assays that we had initially included in the ProteinNPT paper (eg., A0A220GHA5, CAS9 STRP1 negative). We strongly encourage using the latest version of the supervised benchmark in ProteinGym. We already released detailed performance for all baselines on all 217 substitution assays and all CV schemes [here](https://github.com/OATML-Markslab/ProteinGym/blob/main/README.md?plain=1#L92). We are also compiling all predictions for all mutants for all baselines and will be releasing that soon.

Best,
Pascal
